curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-asynchronous-programming-by-building-an-fcc-forum-leaderboard/64649b108df035051cb2ba2d.md

```diff
@@ -16,15 +16,22 @@ And with those changes, your freeCodeCamp forum leaderboard project is now compl
 Your opening anchor tag should have a `target` attribute set to `_blank`.
 
 ```js
-assert.match(code, /<a\s+(?:class=['"]post-title['"]\s+target=['"]_blank['"]|target=['"]_blank['"]\s+class=['"]post-title['"])(?:.|\n)*?<\/a>/);
+assert.match(
+  code,
+   /<a\s+(?:[^>]*\s+)?target=['"]_blank['"](?:\s+[^>]*)?>.*<\/a>/
+);
 ```
 
-Your opening anchor tag should have a `href` attribute set to `${forumTopicUrl}${slug}/${id}` after the `target` attribute.
+Your opening anchor tag should have a `href` attribute set to `${forumTopicUrl}${slug}/${id}`.
 
 ```js
-assert.match(code, /<a\s+class=['"]post-title['"]\s+target=['"]_blank['"]\s+href=['"]\$\{forumTopicUrl\}\$\{slug\}\/\$\{id\}['"]\s*>([\s\S]*?)<\/a>/);
+assert.match(
+  code,
+  /<a\s+(?:[^>]*\s+)?href=['"]\${forumTopicUrl}\${slug}\/\${id}['"](?:\s+[^>]*)?>.*<\/a>/
```

Comment:
```suggestion
  /<a\s+(?:[^>]*\s+)?href\s*=\s*('|")\${forumTopicUrl}\${slug}\/\${id}\1(?:\s+[^>]*)?>.*<\/a>/s
```

Reply:
@gikf I refined the regex as mentioned allowing optional spaces in the new PR